### ISSUE TITLE
Failed to unpack

### ISSUE THREAD

**Issue by dnabanita7** [fonsp/Pluto.jl#1421]
I am using Mozilla Firefox the latest one and when I do ``Pluto.run()`` from Julia REPL, it shows this error or tries reconnecting frequently.
```
Failed to unpack message
Error: BUFFER_SHORTAGE

{"isTrusted":true}
```

This might be because I am trying to get StatsBase and StatsKit in the workflow. This is taking more time than usual.

**Comment by dnabanita7** [comment]
I am also frequently getting the following error:
```
┌ Error: Exception while generating log record in module Pluto at /home/nabanita/.julia/packages/Pluto/HmZoH/src/webserver/WebServer.jl:161
│   exception =
│    UndefVarError: parentbody not defined
│    Stacktrace:
│     [1] macro expansion
│       @ ./logging.jl:341 [inlined]
│     [2] (::Pluto.var"#302#312"{Pluto.ServerSession, Base.RefValue{Function}})(clientstream::HTTP.WebSockets.WebSocket{HTTP.ConnectionPool.Transaction{Sockets.TCPSocket}})
│       @ Pluto ~/.julia/packages/Pluto/HmZoH/src/webserver/WebServer.jl:161
│     [3] upgrade(f::Pluto.var"#302#312"{Pluto.ServerSession, Base.RefValue{Function}}, http::HTTP.Streams.Stream{HTTP.Messages.Request, HTTP.ConnectionPool.Transaction{Sockets.TCPSocket}}; binary::Bool)
│       @ HTTP.WebSockets ~/.julia/packages/HTTP/D0FSE/src/WebSockets.jl:160
│     [4] upgrade
│       @ ~/.julia/packages/HTTP/D0FSE/src/WebSockets.jl:142 [inlined]
│     [5] (::Pluto.var"#301#311"{Pluto.ServerSession, Base.RefValue{Function}, HTTP.Handlers.Router{Symbol("##257")}})(http::HTTP.Streams.Stream{HTTP.Messages.Request, HTTP.ConnectionPool.Transaction{Sockets.TCPSocket}})
│       @ Pluto ~/.julia/packages/Pluto/HmZoH/src/webserver/WebServer.jl:135
│     [6] handle
│       @ ~/.julia/packages/HTTP/D0FSE/src/Handlers.jl:270 [inlined]
│     [7] #4
│       @ ~/.julia/packages/HTTP/D0FSE/src/Handlers.jl:346 [inlined]
│     [8] macro expansion
│       @ ~/.julia/packages/HTTP/D0FSE/src/Servers.jl:413 [inlined]
│     [9] (::HTTP.Servers.var"#13#14"{HTTP.Handlers.var"#4#5"{HTTP.Handlers.StreamHandlerFunction{Pluto.var"#301#311"{Pluto.ServerSession, Base.RefValue{Function}, HTTP.Handlers.Router{Symbol("##257")}}}}, HTTP.ConnectionPool.Transaction{Sockets.TCPSocket}, HTTP.Servers.Server{Nothing, Sockets.TCPServer}, HTTP.Streams.Stream{HTTP.Messages.Request, HTTP.ConnectionPool.Transaction{Sockets.TCPSocket}}})()
│       @ HTTP.Servers ./task.jl:411
└ @ Pluto ~/.julia/packages/Pluto/HmZoH/src/webserver/WebServer.jl:161
```

**Comment by fonsp** [comment]
Both errors seem to point to the websocket stream read not working. I added commit that fixes https://github.com/fonsp/Pluto.jl/issues/1421#issuecomment-907965798 and provides better logging in that case, but we have to wait until it happens again.

https://github.com/colyseus/colyseus.js/issues/1 seems to suggest that the WS connection closed.

@dnabanita7 What OS are you using? Are you running Pluto on the same computer or are you using a proxy?

**Comment by dnabanita7** [comment]
I use Linux Ubuntu one. Nope, use it on the same system and don't use a proxy.

**Comment by fonsp** [comment]
This has hopefully been fixed by #2185
